tests/modules/token_embedders/pretrained_transformer_mismatched_embedder_test.py

```diff
@@ -143,3 +143,43 @@ def test_token_without_wordpieces(self):
         assert not torch.isnan(bert_vectors).any()
         assert all(bert_vectors[0, 1] == 0)
         assert all(bert_vectors[1, 1] == 0)
+
+    def test_exotic_tokens_no_nan_grads(self):
+        token_indexer = PretrainedTransformerMismatchedIndexer("bert-base-uncased")
+
+        sentence1 = ["A", "", "AllenNLP", "sentence", "."]
+        sentence2 = ["A", "\uf732\uf730\uf730\uf733", "AllenNLP", "sentence", "."]
+
+        tokens1 = [Token(word) for word in sentence1]
+        tokens2 = [Token(word) for word in sentence2]
+        vocab = Vocabulary()
+        params = Params(
+            {
+                "token_embedders": {
+                    "bert": {
+                        "type": "pretrained_transformer_mismatched",
+                        "model_name": "bert-base-uncased",
+                    }
+                }
+            }
+        )
+        token_embedder = BasicTextFieldEmbedder.from_params(vocab=vocab, params=params)
```

Comment:
Can you just make this:
```suggestion
        token_embedder = BasicTextFieldEmbedder({"bert": PretrainedTransformerMismatchedEmbedder("bert-base-uncased")})
```
No need to use `Params` here.  (be sure to run black on that line, it might be too long, and this might require adding some imports above)

Reply:
Changed.